packages/esm-patient-appointments-app/src/appointments/appointments-form.scss

```diff
@@ -0,0 +1,70 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vars";
+@import "~carbon-components/src/globals/scss/mixins";
+@import "~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/styles";
+@import "~carbon-components/src/globals/scss/vendor/@carbon/layout/scss/generated/spacing";
+
+.productiveHeading02 {
+  color: $color-gray-70;
+  @include carbon--type-style("productive-heading-02");
+}
+
+$openmrs-background-grey: #ededed;
```

Comment:
Not 100% sure what the decision was here - should such aliases be added to the individual SCSS files or globally in the styleguide? 

CC @FlorianRappl

Reply:
This colour isn't provided yet in the colour palette, once this[PR] (https://github.com/openmrs/openmrs-esm-core/pull/184) is merged will remove this locally defined colour

Reply:
Not 100% sure either. I tried addressing this in https://github.com/openmrs/openmrs-esm-core/pull/184, but I'm not really sure how to move that forward.